src/autoplex/auto/phonons/flows.py

```diff
@@ -273,7 +273,7 @@ def make(
 
         for structure, mp_id in zip(structure_list, mp_ids):
             self.supercell_settings.setdefault(mp_id, {})
-            logging.info(
+            logging.warning(
```

Comment:
Is this something user should be worrying about ? Is there a way to not have it same?

Reply:
There is no way of having it not the same (apart from calling the modules indivually and building an own wf), and that's what I want the user to be warned about.